Assets/_BForBoss/Scripts/Debug/DebugWindow.cs

```diff
@@ -0,0 +1,172 @@
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace BForBoss
+{
+    public class DebugWindow : MonoBehaviour
+    {
+
+        private const Key _keyCodeCharacter = Key.F1;
```

Comment:
```suggestion
        private const Key _keyCodeCharacter = Key.Backquote;
        private const Key _keyCodeModifier = Key.LeftCtrl;
```

Reply:
The thing is with LeftCTrl is that the player will crouch before the time Pauses. This might not be wanted by the user? What do you think?

Reply:
so 2 options
1. it's for explicitly for the devs, so UX is not my top priority (and its like the only option between Mac and Windows which is common and not super weird)
2. just use the Backquote - ` key.

Reply:
hmm, we can also create a command specifically for Mac and then use `#If Unity_MAC` pragma, if you want to use the modifier/character combo

Reply:
Im also okay using backquote too for now

Reply:
just easier to backquote. but it's ultimately up to you.